docs/common/craft-parts/reference/plugins/make_plugin.rst

```diff
@@ -0,0 +1,64 @@
+.. _craft_parts_make_plugin:
+
+Make plugin
+===========
+
+The Make plugin can be used with projects that use a ``Makefile`` to
+build with `GNU Make`_. After a successful build, this plugin will run
+the ``install`` ``Makefile`` target with ``DESTDIR`` set to
+``$CRAFT_PART_INSTALL``.
+
+Keywords
+--------
+
+In addition to the common :ref:`plugin <part-properties-plugin>` and
+:ref:`sources <part-properties-sources>` keywords, this plugin provides the following
+plugin-specific keywords:
+
+make-parameters
+~~~~~~~~~~~~~~~
+**Type:** list of strings
+**Default:** []
+
+Options to pass to make.
+
+
+Dependencies
+------------
+
+The plugin requires ``make``, its installation is handled by the
+plugin itself.
+
+From the project, the ``Makefile`` must support the ``install`` target
+and the use of ``DESTDIR``.
+
+
```

Comment:
The build is also affected by the parallel build count, but apparently this is handled by the application.